unit-testing/test/UnitTesting/Assertions.cs

```diff
@@ -0,0 +1,85 @@
+using System.Dynamic;
+
+namespace UnitTesting;
+
+public class Assertions : Spec

```

Comment:
a custom should extension is missing, sample from do :point_down: 
 https://github.com/mouseless/do/blob/461321f134ec6cc5660eb36ea27ba60695c834b8/test/core/Do.Test.Architecture/ArchitectureSpecExtensions.cs#L72